src/gc/sample/gcenv.ee.cpp

```diff
@@ -200,13 +203,15 @@ bool GCToEEInterface::TrapReturningThreads()
     return !!g_TrapReturningThreads;
 }
 
-gc_alloc_context * GCToEEInterface::GetAllocContext(Thread * pThread)
+gc_alloc_context * GCToEEInterface::GetAllocContext()
 {
+    Thread* pThread = ::GetThread();
     return pThread->GetAllocContext();
```

Comment:
It seems like `::GetThread()` may return `nullptr` so it would be good to add `assert(pThread != nullptr);`

Reply:
All of these are in the sample. The only way GetThread will be null is if we call it from a server GC thread, which the sample currently doesn't support.

Reply:
I missed it. Sorry.